deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
Is there anything that needs change in this PR? AFAICT this file is always included after the `Make.inc` so the `LLVM_CONFIG_HOST` is always set and available at this point for `USE_SYSTEM_LLVM`.